sdmetrics/reports/utils.py

```diff
@@ -624,3 +625,75 @@ def print_results_for_level(out, results, level):
         out.write(f'\n{level}:\n')
         for result in results[level]:
             out.write(f'{level_marks[level]} {result}\n')
+
+
+def _validate_categorical_values(real_data, synthetic_data, metadata, table=None):
+    """Get categorical values found in synthetic data but not real data for all columns.
+
+    Args:
+        real_data (pd.DataFrame):
+            The real data.
+        synthetic_data (pd.DataFrame):
+            The synthetic data.
+        metadata (dict):
+            The metadata.
+        table (str, optional):
+            The name of the current table, if one exists
+    """
+    if table:
+        warning_format = 'Unexpected values ({values}) in column "{column}" '\
+                         f'and table "{table}"'
```

Comment:
In general when having to deal with this situations I would recommend using `(`. Also looks like missing formatting for `values` and `column` (f-string).